### PR TITLE
feat: add compatibility with transifex

### PR DESCRIPTION
**Pull request by regisb**
Now that the frontend-app-gradebook resource was added to Transifex, we can
start pulling strings fromt there. For now, the project contains very few
translated string.

Strings were pulled by running: make pull_translations

This is for https://github.com/openedx/build-test-release-wg/issues/107

**TL;DR -**

Add compatibility with transifex.

**What changed?**

- Updated translation strings from Transifex
- Updated git-ignored files 

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

- Setup the site language to French in the user account
- Verify that at least some of the strings are translated when accessing the gradebook.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta
CC: @sarina 

This PR will address https://github.com/openedx/build-test-release-wg/issues/107

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [OSPR-6233](https://openedx.atlassian.net/browse/OSPR-6233) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: src/i18n/messages/ar.json [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
+  "gradebook.GradesView.EditModal.title": "Edit Grades",
+  "gradebook.GradesView.EditModal.closeText": "Cancel",
+  "gradebook.GradesView.EditModal.contactSupport": "Showing most recent actions (max 5). To see more, please contact support",
+  "gradebook.GradesView.EditModal.saveVisibility": "Note: Once you save, your changes will be visible to students.",
+  "gradebook.GradesView.EditModal.saveGrade": "Save Grades",
+  "gradebook.GradesView.EditModal.Overrides.adjustedGradeHeader": "Adjusted grade",
+  "gradebook.GradesView.EditModal.Overrides.dateHeader": "Date",
+  "gradebook.GradesView.EditModal.Overrides.graderHeader": "Grader",
+  "gradebook.GradesView.EditModal.Overrides.reasonHeader": "Reason",
+  "gradebook.GradesTab.usersVisibilityLabel'": "Showing {filteredUsers} of {totalUsers} total learners",
```
Comment (sarina) [inline src/i18n/messages/ar.json:38]: I know this PR is not focused on the actual string definitions, but this one is particularly egregious. `filteredUsers` as a variable implies a list of users, but contextually I think it must be a number. `Showing {numFilteredUsers} of {numTotalUsers}` would be significantly clearer. Flagging so team engineers can understand.

File: src/i18n/messages/ar.json [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+  "gradebook.GradesView.editSuccessAlert": "The grade has been successfully edited.  You may see a slight delay before updates appear in the Gradebook.",
+  "gradebook.GradesView.maxCourseGradeInvalid": "Maximum course grade must be between 0 and 100",
+  "gradebook.GradesView.minCourseGradeInvalid": "Minimum course grade must be between 0 and 100",
+  "gradebook.GradesTab.FilterBadges.assignment": "Assignment",
+  "gradebook.GradesTab.FilterBadges.assignmentGrade": "Assignment Grade",
+  "gradebook.GradesTab.FilterBadges.assignmentType": "Assignment Type",
+  "gradebook.GradesTab.FilterBadges.cohort": "Cohort",
+  "gradebook.GradesTab.FilterBadges.courseGrade": "Course Grade",
+  "gradebook.GradesTab.FilterBadges.includeCourseRoleMembers": "Include Course Team Members",
+  "gradebook.GradesTab.FilterBadges.track": "Track"
```
Comment (sarina) [inline src/i18n/messages/ar.json:72]: Track has a few English definitions, this needs a translator's comment.

File: src/i18n/messages/ar.json [hunk 3]
```diff
@@ -1,2 +1,73 @@
...
+  "gradebook.GradesView.EditModal.Overrides.adjustedGradeHeader": "Adjusted grade",
+  "gradebook.GradesView.EditModal.Overrides.dateHeader": "Date",
+  "gradebook.GradesView.EditModal.Overrides.graderHeader": "Grader",
+  "gradebook.GradesView.EditModal.Overrides.reasonHeader": "Reason",
+  "gradebook.GradesTab.usersVisibilityLabel'": "Showing {filteredUsers} of {totalUsers} total learners",
+  "gradebook.GradesView.editFilterLabel": "Edit Filters",
+  "gradebook.GradesView.table.headings.email": "Email",
+  "gradebook.GradesView.table.headings.totalGrade": "Total Grade (%)",
+  "gradebook.GradesView.table.headings.username": "Username",
+  "gradebook.GradesView.table.labels.studentKey": "Student Key*",
```
Comment (sarina) [inline src/i18n/messages/ar.json:43]: Is the `*` required? Needs a translators comment

File: src/i18n/messages/ar.json [hunk 4]
```diff
@@ -1,2 +1,73 @@
 {
-}
+  "gradebook.BulkManagementHistoryView.heading": "Bulk Management History",
```
Comment (sarina) [inline src/i18n/messages/ar.json:2]: What des this mean? Needs a pretty decent translator's comment.

**Comment by sarina** [comment]
@regisb don't worry about my comments, I was just frustrated at the lack of attention and shouldn't have taken it out on your PR. I'm going to leave them up, if that's OK, because I've asked @Carlos-Muniz to dive into i18n as a starting task and I'd like him to investigate/fix.

**Review by sarina — APPROVED** [review]
Absolutely

**Comment by edx-webhook** [comment]
@regisb 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-gradebook&entry.867055334=https://github.com/edx/frontend-app-gradebook/pull/217&entry.1484655318=https://github.com/regisb&entry.752974735=2021-11-22+11:22&entry.1917517419=2021-11-29+17:37&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by natabene** [comment]
@regisb @sarina Thank you both!

**Pull request merged**